packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -345,7 +345,16 @@ const BaseButton = ({
       motionEasing={motionEasing}
     >
       {isLoading ? (
-        <Box display="flex" justifyContent="center">
+        <Box
+          display="flex"
+          justifyContent="center"
+          alignItems="center"
+          position="absolute"
+          top={0}
+          left={0}
+          bottom={0}
+          right={0}
```

Comment:
Hope you tested on web.

Reply:
Attached screenshot

Reply:
Oh you did.cool!

Reply:
Also, check the story https://61c19ee8d3d282003ac1d81c-cfzrfrubtk.chromatic.com/?path=/story/components-button--button-loading&globals=measureEnabled:false

Reply:
Also, check the story https://61c19ee8d3d282003ac1d81c-cfzrfrubtk.chromatic.com/?path=/story/components-button--button-loading&globals=measureEnabled:false